src/styles/watch.scss

```diff
@@ -0,0 +1,65 @@
+.watch {
+  width: 700px;
+  height: 600px;
+  background-color: antiquewhite;
+  margin: auto;
+  position: relative;
+
+  &__hour,
+  &__minute,
+  &__second {
+    border-radius: 10px;
+    box-shadow: 0 5px 10px rgb(100, 100, 100);
+    position: absolute;
+    bottom: 50%;
+  }
+
+  &__hour {
+    left: $left-hour;
+
+    width: $width-hour;
+    height: 90px;
+    background-color: rgb(13, 13, 13);
+
+    animation: moving ($time-min*60) infinite steps(60);
+    transform-origin: center 105px;
+  }
+
+  &__minute {
+    left: $left-min;
+
+    background-color: rgb(255, 0, 0);
+    width: $width-min;
+    height: 175px;
+
+    animation: moving $time-min infinite steps(60);
+    transform-origin: center 190px;
+  }
+
+  &__second {
+    left: $left-sec;
+
+    width: $width-sec;
+    height: 250px;
+    background-color: rgb(0, 255, 0);
+
+    animation: moving ($time-min/60) infinite steps(60);
+    transform-origin: center 265px;
+  }
+
+}
+
+.point {
+  box-sizing: border-box;
+  height: $width-point;
+  width: $width-point;
+  margin: auto;
+  text-align: center;
+  align-items: center;
+  border-radius: 50%;
+  border: 3px solid #000;
+  background-color: rgb(242, 255, 0);
+  position: absolute;
+  top: 50%;
+  left: 50%;
```

Comment:
- it's not centered. try ($watch-size / 2) - ($point-size / 2)
- you don't need text-align, align-items, margin

Reply:
You can try Natalia's solution for centering, it would be cleaner probably :) and remove unnecessary properties